### PR TITLE
removed more jsoc tests

### PR DESCRIPTION
**Pull request by nabobalis**
I also got:

```
The 'asyncio_mode' default value will change to 'strict' in future, please explicitly use 'asyncio_mode=strict' or 'asyncio_mode=auto' in pytest configuration file.:DeprecationWarning
```
So I patched that.

### CODE REVIEW

File: .github/workflows/cancel_ci.yml [hunk 1]
```diff
@@ -14,3 +14,6 @@ jobs:
       uses: styfle/cancel-workflow-action@0.9.1
       with:
         workflow_id: ${{ github.event.workflow.id }}
+        ignore_sha: true
+        all_but_latest: true
+        access_token: ${{ github.token }}
```
Comment (nabobalis) [inline .github/workflows/cancel_ci.yml:19]: This one isn't needed but yolo.

**Comment by nabobalis** [comment]
Online fails are now HEC.

File: sunpy/net/jsoc/tests/test_jsoc.py [hunk 2]
```diff
@@ -15,6 +14,7 @@
 from sunpy.net.jsoc import JSOCClient, JSOCResponse
 from sunpy.util.exceptions import SunpyUserWarning
 
+TEST_EMAIL = "nabil.freij@gmail.com"
```
Comment (wtbarnes) [inline sunpy/net/jsoc/tests/test_jsoc.py:17]: can you just put this in conftest so we don't have to change it everywhere if we want to change it to something less-Nabil-specific later?

**Review by wtbarnes — APPROVED** [review]

File: sunpy/net/jsoc/tests/test_jsoc.py (hunk 2 above)
Comment (nabobalis) [reply sunpy/net/jsoc/tests/test_jsoc.py:17]: Sure.

File: sunpy/conftest.py [hunk 3]
```diff
@@ -33,6 +34,12 @@
 console_logger.setLevel('INFO')
 
 
+@pytest.fixture(scope='session', autouse=True)
+def jsoc_test_email():
+    # JSOC Test email
+    pytest.jsoc_test_email = "nabil.freij@gmail.com"
```
Comment (Cadair) [inline sunpy/conftest.py:40]: Is this really a recommended pattern, to just attach an attribute to an imported module? This seems real gross.

Comment (ConorMacBride) [reply sunpy/conftest.py:40]: I'll change this in a PR

Comment (nabobalis) [reply sunpy/conftest.py:40]: It was more fun this way.

**Pull request merged**